client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-card/index.tsx

```diff
@@ -20,7 +20,13 @@ export default function SiteCard( { rows, columns }: Props ): ReactElement {
 
 	const [ isExpanded, setIsExpanded ] = useState( false );
 
-	const toggleIsExpanded = () => {
+	const toggleIsExpanded = (
```

Comment:
Can we wrap this in `useCallback`?

Reply:
Fixed in: 44f4bc6fad8841a6816d5b7c41c06a8a85a429ea